cdragontoolbox/binfile.py

```diff
@@ -482,7 +482,7 @@ def parse_bintype(self, v):
                 v = 0x80
             elif v >= 19:
                 v = 0x80 + v - 18
-        elif self.btype_version < 1008:
+        if self.btype_version < 1008:
```

Comment:
Why this change?

Reply:
Because early bins failed to parse without it.